hugo.yaml

```diff
@@ -1,4 +1,4 @@
-baseURL: https://docs.wandb.ai
+baseURL: /
```

Comment:
When running locally, this makes it so that the link provided in the terminal when doing `hugo serve` does not work, as it removes the `https://` protocol from the string. Must we do it this way?